wp-content/themes/citylimits/inc/widgets/class-citylimits-special-projects-featured-content-widget.php

```diff
@@ -191,6 +208,9 @@ function form( $instance ) {
 			'num_sentences' => 2,
 			'show_byline' => '',
 			'hide_byline_date' => '',
+			'cat' => 0,
+			'tag' => '',
+			'author' => '',
```

Comment:
Author is sanitized with `intval()` as an integer, but the default is an empty string. Is that type confusion something we're worried about?

Reply:
If it is, we should probably update it in Largo Recent Posts. 😬 

https://github.com/INN/largo/blob/1744bbc7ae99f501129d158c3b585cddf6b5409e/inc/widgets/largo-recent-posts.php#L148

Have we noticed any issues with it before?

Reply:
I don't think we've had any problems with it before. There's a Largo issue https://github.com/INN/largo/issues/1807 for running code standards tests against all the Largo widgets; we can probably just work this fix into that code cleanup.